### PR TITLE
Add remaining prop examples for the Image component

### PR DESCRIPTION
**Pull request by JosephSemrai**
## Summary

Adds missing prop examples for the Image component in rn-tester.

## Changelog
[Internal] [Added] New examples for LoadingIndicatorSource, Accessibility, AccessibilityLabel, fadeDuration, onLayout, onPartialLoad, accessibilityLabel, etc.

## Test Plan

- [x] Build rn-tester and verify that it runs
- [x] Interact with each component
- [x] Verify that the examples function as intended

### CODE REVIEW

**Comment by mkcode** [comment]
Hi @JosephSemrai - Let's fix the flow errors in the analyze code step next.

File: packages/rn-tester/js/examples/Image/ImageExample.js [hunk 1]
```diff
@@ -340,6 +346,199 @@ class MultipleSourcesExample extends React.Component<
...
+
+    return (
+      <View>
+        <View style={{flexDirection: 'row', justifyContent: 'space-between'}}>
+          <Text style={styles.touchableText} onPress={this.reloadImage}>
+            Refresh Image
+          </Text>
+          {/* <Text style={styles.touchableText} onPress={this.increaseImageSize}>
+            Increase image size
+          </Text> */}
```
Comment (rickhanlonii) [inline packages/rn-tester/js/examples/Image/ImageExample.js:386]: Could we remove the commented out code?

File: packages/rn-tester/js/examples/Image/ImageExample.js [hunk 2]
```diff
@@ -340,6 +346,199 @@ class MultipleSourcesExample extends React.Component<
   }
 }
 
+type LoadingIndicatorSourceExampleState = {|
+  imageHash: number,
+|};
+
+type LoadingIndicatorSourceExampleProps = $ReadOnly<{||}>;
+
+class LoadingIndicatorSourceExample extends React.Component<
```
Comment (rickhanlonii) [inline packages/rn-tester/js/examples/Image/ImageExample.js:355]: Is it possible to use hooks instead of classes?

Comment (JosephSemrai) [reply packages/rn-tester/js/examples/Image/ImageExample.js:355]: I made use of classes over functional components for the sake of consistency (as the other, previous examples also use classes).

**Pull request closed without merging**